Dockerfile

```diff
@@ -1,34 +1,45 @@
+FROM openjdk:8-jdk AS build
+MAINTAINER Gabe Conradi <gabe@tumblr.com>
```

Comment:
I'd imagine this shouldn't just be you but a list or something

Reply:
If theres a good mailing list for owners, happy to update. @defect?

Reply:
How about using the google group? collins-sm@googlegroups.com